frontend/src/screens/settings/AutoUnlock.tsx

```diff
@@ -0,0 +1,129 @@
+  const [loading, setLoading] = React.useState(false);
+
+  const onSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+
+    try {
+      setLoading(true);
+      await request("/api/auto-unlock", {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          unlockPassword,
+        }),
+      });
+      await refetchInfo();
+      setUnlockPassword("");
+      toast({
+        title: `Successfully ${unlockPassword ? "enabled" : "disabled"} auto-unlock`,
+      });
+    } catch (error) {
+      toast({
+        title: "Auto Unlock change failed",
+        description: (error as Error).message,
+        variant: "destructive",
+      });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  if (!info) {
+    return <Loading />;
+  }
+  if (!info.autoUnlockPasswordSupported) {
+    return <p>Your Hub does not support this feature.</p>;
+  }
+
+  return (
+    <>
+      <SettingsHeader
+        title="Auto Unlock"
+        description="Configure Alby Hub will automatically unlock on start (e.g. after machine reboot)"
+      />
+      <Container>
+        {!info.autoUnlockPasswordEnabled && (
+          <>
+            <Alert>
+              <AlertTriangle className="h-4 w-4" />
+              <AlertTitle>Security Warning</AlertTitle>
+              <AlertDescription>
+                By enabling this feature your unlock password will be saved in
+                plaintext on the device running this hub. If the device is
+                compromised,{" "}
+                <span className="underline">your funds will be lost</span>.{" "}
+                <span className="font-semibold">
+                  Do not enable this feature if you do not physically own the
+                  machine Alby Hub is running on.
+                </span>
+              </AlertDescription>
+            </Alert>
+            <form
+              onSubmit={onSubmit}
+              className="w-full flex flex-col gap-3 mt-3"
+            >
+              <div className="grid gap-1.5">
+                <Label htmlFor="unlock-password">Unlock Password</Label>
+                <Input
+                  id="unlock-password"
+                  type="password"
+                  name="password"
+                  onChange={(e) => setUnlockPassword(e.target.value)}
+                  value={unlockPassword}
+                  placeholder="Password"
+                />
+              </div>
+
+              <LoadingButton loading={loading}>
+                Enable Auto Unlock
+              </LoadingButton>
+            </form>
+          </>
+        )}
+        {info.autoUnlockPasswordEnabled && (
+          <>
+            <Alert>
+              <AlertTriangle className="h-4 w-4" />
+              <AlertTitle>Security Warning</AlertTitle>
+              <AlertDescription>
+                Your unlock password is currently saved in plaintext on the
```

Comment:
"currently" makes it sound like it's our fault that this is insecure

I would say something like: 

In some situations it can be impractical to manually unlock the wallet every time Alby Hub is started. In those cases you can save the unlock password in plain text to tell the wallet to auto-unlock itself. 
Attention: everyone who has access to your machine could read that password and take your funds. Use this only in a secure environment.

Reply:
Updated and simplified to have this as a constant alert rather than two variants